### DISCUSSION TITLE
About ATOM syntax please

### DISCUSSION THREAD

**Discussion by pitser90** [atom/atom#23437] [Q&A]
Please:
I´m checking a tutorial in UDEMY where they use Jupyter notebook for Python, and for example, when using the data type SERIES of PANDAS... when they write: pd.Series()   ... then, they can press SHIFT TAB and a little window opens showing information about the pd.Series().

Information, as: pd.Series(data = None, dtype=None, name=None, ........etc).... and also gives some information. This can be done with almost all of the built-in functions.

Does anyone know how can I visualize this stuff using ATOM? Thanks so much

**Comment by ThatXliner** [comment]
Seems like you want something like [ide-python](https://github.com/atom-community/ide-python)

**Reply by pitser90** [reply]
thank u! Gonna check that

**Reply by pitser90** [reply]
Would you please know what to do when I get this error in the CMD, after writing this: 
python -m pip install 'python-language-server[all]'

ERROR: Invalid requirement: "'python-language-server[all]'"

**Reply by icecream17** [reply]
The error is saying that `'python-language-server[all]'` is not a valid package name. Although it seems like it should work, what version of python are you using?

**Reply by pitser90** [reply]
I´m using Python 3.9  64-bit.  Also it seems that I need to install ATOM packages. But first I would like the previous one to work.
Thank you!
